hibernate-core/src/test/java/org/hibernate/orm/test/hql/ASTParserLoadingTest.java

```diff
@@ -925,12 +926,20 @@ public void testSelectClauseCaseWithCountDistinct() {
 
 		s.flush();
 
-		Number count = (Number) s.createQuery( "select count(distinct case when bodyWeight > 100 then description else null end) from Human" ).uniqueResult();
-		assertEquals( 1, count.intValue() );
-		count = (Number) s.createQuery( "select count(case when bodyWeight > 100 then description else null end) from Human" ).uniqueResult();
-		assertEquals( 2, count.intValue() );
-		count = (Number) s.createQuery( "select count(distinct case when bodyWeight > 100 then nickName else null end) from Human" ).uniqueResult();
-		assertEquals( 2, count.intValue() );
+		if ( !( getDialect() instanceof InformixDialect && getDialect().getVersion().isBefore( 11, 70 ) ) ) {
```

Comment:
Please use the `@SkipForDialect` annotation for this instead.

Reply:
I will have to rework this test into JUnit 5 because the current deprecated annotation `@SkipForDialect` does not have the attributes `majorVersion`, `minorVersion`.